permissions-policy/payment-extension-allowed-by-permissions-policy-attribute.https.sub.html

```diff
@@ -13,9 +13,7 @@
 
     promise_test(t => {
       return test_feature_availability_with_post_message_result(
-          t, cross_origin_src, "NotSupportedError#The 'payment' or 'publickey-credentials-create' features are not " +
-          "enabled in this document. Permissions Policy may be used to " +
-          "delegate Web Payment capabilities to cross-origin child frames.");
+          t, cross_origin_src, "NotSupportedError");
     }, feature_name + ' is not supported in cross-origin iframe without ' + header);
```

Comment:
Could we change the `feature_name` to be `'WebAuthn "payment" extension'` to make it more clear what this test is about?